distributed/tests/test_asyncprocess.py

```diff
@@ -79,7 +79,7 @@ async def test_simple():
     t1 = time()
     await proc.join(timeout=0.02)
     dt = time() - t1
-    assert 0.2 >= dt >= 0.01
+    assert 0.2 >= dt >= 0.001
```

Comment:
This seems fine. It may be possible to go futher: `assert 0.2 >= dt >= 1e-5` or `assert 0.2 >= dt >= 1e-9` for example?

Reply:
Yup.  I don't know enough here to want to dive in too deeply.  This is just a drive-by contribution to hopefully reduce the pain around CI.

Reply:
To add more color here, the person who wrote this test clearly wanted there to be some non-zero value here.  Unfortunatley I don't know exactly what they intended.  Probably `.join` shouldn't be a no-op, a process actually needs to clean up, and they want to verify that.  Going all the way to an extreme like `1e-9` is probably too strong of a statement.

My goal here is to get CI working while not getting sucked too deeply into this issue (I want to stay focused).  In order to achieve that goal I want to make the least controversial change I can make, while still probably solving most of the CI issue.

Switching down one order of magnitude seems minimally controversial.  I don't want to go farther than that, in case that would trigger a longer discussion about exactly what we should do with AsyncProcess (which seems like a distraction outside of my current objectives).

Reply:
@mrocklin Got it. BTW I approved, so I wasn't requesting further decreases in magnitude, but I was saying that it *might* be possible. This is good to merge I think.